workflow/controller/controller.go

```diff
@@ -262,6 +284,56 @@ func (wfc *WorkflowController) createSynchronizationManager() error {
 	return nil
 }
 
+func (wfc *WorkflowController) runConfigMapWatcher(stopCh <-chan struct{}) {
+	retryWatcher, err := apiwatch.NewRetryWatcher("1", &cache.ListWatch{
```

Comment:
Watcher is watching all configmap across namespace, We don't know the configmap name to get the version, so initialResourceVersion =1